docs/pages/includes/config-reference/instance-wide.yaml

```diff
@@ -55,10 +55,18 @@ teleport:
         # token_name: /var/lib/teleport/tokenjoin
         token_name: "token-name"
 
-    # Optional CA pin of the auth server. This enables a more secure way of
-    # adding new nodes to a cluster. See "Adding Nodes to your Cluster"
-    # (https://goteleport.com/docs/setup/admin/adding-nodes).
-    ca_pin: "sha256:7e12c17c20d9cb504bbcb3f0236be3f446861f1396dcbb44425fe28ec1c108f1"
+    # Optional CA pin of the Auth Service, which enables new agents to trust a
+    # Teleport cluster when joining via the Auth Service directly. This can be
+    # either the literal value of the CA pin or an absolute path to a file
```

Comment:
```suggestion
    # as a literal value of the CA pin or by specifyng the absolute path to a file that contains the CA pin value.
```

Reply:
I've made a suggestion that makes this a bit more concise—let me know what you think!

Reply:
Not sure why you want to remove the actor from the sentence (someone is make the decision and typing the info), but if you want to keep it this way, I'd suggest identifying what "This" refers to, maybe something like:
This [field? setting?] can be the literal value of the CA pin or the absolute path to a file that contains the CA pin value.

Reply:
Would you mind quoting the part of the text you're referring to in your last comment? The new text does not use the word "this", so I'm not sure if the last comment has to do with my latest changes.

Reply:
Sorry about that-you're right. I was trying to parse the diff text instead of viewing the file directly. The latest changes look fantastic. 👍